bitnami/jenkins/values.yaml

```diff
@@ -40,6 +40,9 @@ nameOverride: ""
 fullnameOverride: ""
 ## @param commonLabels Labels to add to all deployed objects
 ##
+replicas: {}
```

Comment:
Comment has to be on top of the key.
Define it like here: https://github.com/bitnami/charts/blob/main/bitnami/apache/values.yaml#L108